### ISSUE TITLE
Integrate flow based style

### ISSUE THREAD

**Issue by lawrennd** [lawrennd/lamd#5]
Switch these command line capabilities to be two stage. 

The first stage is to create a relevant interface file (see e.g. https://github.com/lawrennd/referia/blob/main/referia/config/interface.py) of the form

```
input:
# Specify the input files here

output:
# specify the output files here

compute: 
# Specify the computations that are done on the two
```
Write that file, register it with the publish and subscribe service and then execute the command line on the file directly.

**Comment by lawrennd** [comment]
## Comparison of original request vs actions taken (2026-08-19)

Issue asked for a two-stage CLI: write an `input`/`output`/`compute` interface, register it with a pub/sub service, then execute from that file.

### Done (related, not sufficient)

- `lamd.config.interface.Interface` exists as a thin subclass of lynguine's `Interface` (`_lamd.yml`).
- `mdlist` already runs `CustomDataFrame.from_flow(interface)`, but the interface is mutated in memory from CLI args (`filename`, `type`, `index`) and is never written out.
- CIP-0008 (Closed) added lynguine server sessions for `mdfield` (`--use-server` / `LAMD_USE_SERVER`). That is runtime reuse, not generate-then-execute.
- REQ-0003 extracted the WHAT (transparent, traceable workflows).

### Designed, not implemented

- CIP-0006 (Accepted 2026-01-04) is the HOW for generate + execute: new `lamd` CLI with `talk`/`cv` subcommands, `--generate` / `--execute`, legacy `maketalk`/`makecv` unchanged.
- Phase 1 was decomposed on 2026-02-03 into six backlog tasks. All remain `Proposed`. There is no `lamd/cli.py` and no `lamd` console script in `pyproject.toml`.

### Still a design gap (now a CIP)

- "Register with the publish and subscribe service": no pub/sub broker exists in lamd, lynguine, or referia. CIP-0006 deferred this to Phase 2.
- **CIP-0011 (Proposed)** is the HOW for that sentence: do not invent a bus. The interface file is the published contract; Make/CI/git and optional lynguine sessions are the subscribers. Blocked on CIP-0006 Phase 1 actually emitting files.

### Suggested next steps

1. Review CIP-0011 (Proposed). Accept, reject, or amend.
2. If the two-stage CLI is still wanted, start CIP-0006 Phase 1 (`lamd` entrypoint + interface schema + `lamd talk`). Do not create further backlog until that is in progress.
3. Keep this issue open until generate + execute work; then close it with CIP-0011 (once accepted and documented).

**Comment by lawrennd** [comment]
Correction on placement: CIP-0011 should not live in lamd, and treating “no bus exists” as “don’t build one” was the wrong HOW.

`Interface` already defines connections to other black-box processes. Publish/subscribe of **named flows** is lynguine infrastructure — a registry on top of `from_flow` / `to_flow`, not a message broker and not “the YAML path is the topic.”

- Rejected: [lamd CIP-0011](https://github.com/lawrennd/lamd/blob/main/cip/cip0011.md)
- Proposed: [lynguine CIP-000C](https://github.com/lawrennd/lynguine/blob/main/cip/cip000C.md) and [lynguine REQ-000C](https://github.com/lawrennd/lynguine/blob/main/requirements/req000C_named-flow-composition.md)
- Lamd CIP-0006 generate/execute **consumes** that registry (`publish.name` / `source.type: flow`, then `register_interface`)
